daft/expressions/expressions.py

```diff
@@ -1580,7 +2160,23 @@ def substr(self, start: int | Expression, length: int | Expression | None = None
             If `length` is not provided, the substring will include all characters from `start` to the end of the string.
 
         Example:
-            >>> col("x").str.substr(2, 2)
+            >>> import daft
+            >>> df = daft.from_pydict({"x": ["foo", "bar", "baz"]})
+            >>> df = df.select(df["x"].str.substr(2,2))
```

Comment:
Lets have a substr example thats more than just 1 char, maybe `substr(2,4)` and you could have one string that's only 3 chars long to show that it gets cut off